src/cmd_processor.cc

```diff
@@ -0,0 +1,46 @@
+#include "cmd_processor.h"
+
+#include <libqalculate/Calculator.h>
+
+namespace bb
+{
+    class CMDProcessor::Impl
+    {
+    public:
+        Impl();
+        std::string process(const std::string &cmd);
+
+        Calculator *calc;
```

Comment:
This pointer should be a std::unique_ptr

Reply:
yep! 2d624b1